changelog/57529.fixed

```diff
@@ -0,0 +1,3 @@
+Improves performance. Pofiling `test.ping` on Windows shows that 13 of 17 
```

Comment:
is this a misspelling?

Reply:
Which word? It looks good to me.

Reply:
Pofiling should be Profiling right?

Reply:
Thanks, @bryceml, Pofiling should be Profiling!

Reply:
Corrected